subprojects/execution/src/main/java/org/gradle/internal/execution/UnitOfWork.java

```diff
@@ -102,6 +102,13 @@ interface WorkOutput {
          */
         @Nullable
         Object getOutput();
+
+        /**
+         * Whether the outputs can be reused by other builds.
```

Comment:
:x: That's not true. We still allow reusing the outputs in future builds via incremental build (=up-to-date tasks).

Reply:
Done in 6f1d0728d6f4ea6ab55a751949e6298756aa51b9.